fix(content): Add landing message to novas (#10624)
endless-sky/endless-sky: data/_ui/landing messages.txt

	"planet/browndwarf-t-rogue"
	"planet/browndwarf-y"
	"planet/browndwarf-y-rogue"

> "landing message" "You cannot land on a nova!"

Author: ```suggestion
"landing message" "You cannot land on a supernova remnant!"
```

Author: Let's just avoid this issue entirely.
```suggestion
"landing message" "This environment is too hostile for your ship to land in."
```

Author: It's a "stellar object". When the player moves slowly over a stellar object and presses the landing key, it is interpreted as an attempt to land on that object.

This object can be a planet, star, or anything else. Some of them are landable, some of them are not.

In this case, the entire gas cloud + central body is part of the same (star) stellar object. You can only try to land near the middle of it due to the radius of stars being limited to 80 pixels. So this is not a matter of having a central, landable body; it's just how the limited radius works out to something similar.

	"star/nova-small"
	"star/nova"
	"star/nova-old"
	"star/nova-core"
	"star/nova-old-core"

> 	"star/wr"

Reviewer: It doesn't look that similar; both sprites are pretty different from one another. The nova is using yellow-red-brown colors, and has more uniform jets spurting out from the central star. The WR on the other hand is using very vibrant purple-blue hues, and has more solid designs such as the stretches of energy and the boundary for the outer cloud layer. I don't think what's described here is enough to consider a new sprite for them, they look pretty good in-game.

As an aside, I'm not sure how explicit we need to be about landing on an object. All of these phenomena still stem from a star, so that's not entirely incorrect. I feel the more explanative/specific we are about elements like this, the rest of the game kind of begins to falter because it's very loose in these terms - for example, why is the game telling you you can't land on a planet if it uses the same sprite another rock planet that is landable elsewhere?

> 	"star/wr"

Reviewer: They're smaller spheres, but the spheres also have some differences. The star in the nova is very bright and much more solid. The one in the WR is more transparent, dimmer, and within it has some of the purple hue, which helps imply the ejection a bit more (in my opinion), whereas the brightness of the nova is much more intense and explosive. Though I could see maybe changing the size slightly, if the assets allow that.

> 	"star/wr"

Reviewer 2: ```suggestion
```
